src/components/cards/CountBox.less

```diff
@@ -1,4 +1,4 @@
-@import '../../css/base.less';
+@import '../../../node_modules/@gen3/ui-component/dist/css/base.less';
```

Comment:
I don't think we want to import directly from `node_modules`, can it just be `@import '@gen3/ui-component/...'` ?